test/src/test/java/org/locationtech/geowave/test/services/GeoServerIngestIT.java

```diff
@@ -87,7 +87,10 @@
           // loaders so the RocksDB "singleton" instances are not shared in
           // this JVM and GeoServer, for file-based geoserver data sources, using the REST
           // "importer" will be more handy than adding a layer by referencing the local file system
-          GeoWaveStoreType.ROCKSDB},
+          GeoWaveStoreType.ROCKSDB,
+          // filesystem sporadically fails with a null response on spatial-temporal subsampling
+          // (after the spatial index is removed and the services restarted)
```

Comment:
Will this happen enough to cause frequent build failures on CI?

Reply:
It never happens for me locally but is consistently happening on travis for some reason, I'm assuming its related to the secondary index IT tests failing deletion with both indices, but seems strange to not ever happen locally.